lib/maths/unittest/Makefile

```diff
@@ -88,6 +88,7 @@ SRCS=\
 	CSamplingTest.cc \
 	CSeasonalComponentTest.cc \
 	CSeasonalComponentAdaptiveBucketingTest.cc \
+    CSeasonalTimeTest.cc \
```

Comment:
nit: the indenting is inconsistent

Reply:
See [2e9d47c](https://github.com/elastic/ml-cpp/pull/821/commits/2e9d47cd0955a5a8c22509e8c239e50438b3c5e0)